submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+But what about the risks?
+
+ 
+
+## Nuclear Safety in Factorio: A Brief History
+
+Nucelar power was something that Wube wanted to get around to for quite a while and they shared some early design ideas before releasing the version know in 0.15. When they shared their first implementation tinkers in [FFF#164](https://www.factorio.com/blog/post/fff-164), there was a lot of focus on heating and cooling water and steam. The early designs involved high-temperature steam being cooled and only partially consumed in turbines, then condensed back into water at 95C in cooling towers, and then being fed back into heat exchangers. You can imagine this being difficult to balance, leading perhaps to one thing or another getting stuck pretty quickly. After a few more weeks of working on it, Wube decided around [FFF#167](https://www.factorio.com/blog/post/fff-167) that dealing with so much temperature management can be avoided and that water and steam could simply be treated as two separate fluids with fixed temperatures, hence simplifying regular boilers too. 
+
+Meanwhile, the effects of temperature on the reactor itself were also softened: At first it was hinted that a nuclear reactor that heated up uncontrollably would explode, hence reactors would demand constant monitoring as in real life. This idea was softened before release though, such that a nuclear plants uses its internal emergency cooling systems to never let it itself heat up beyond 1000C, effectively allowing you to safely leave it alone. Well, mostly. Beware that it still *can* explode if is at over 900C and gets destroyed by a reckless player or biter. And if other reactors are close by, this may lead to [quite a light show](https://www.youtube.com/watch?v=l4ePIuhbiPg&ab_channel=Trupen). Despite the trivialized safety, we can see that Wube has provided a nuclear system which could still get pretty interesting and risky via the added complexity from the neighbor bonus. For those who would have preferred the high-risk and more realistic approach, there are multiple [mods]() that offer a look into the problems of meltdowns and radiation hazards.
+
+ Now that we' seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.
+
+## **Common Concerns with Nuclear Power**

```

Comment:
General remarks about the paragraph: I feel like addressing all problems with a single sentence is not enough. I like the 1st point being described, being in appendix or in the text itself, I don't know, while the 3 next ones seems more like throwing an idea in the debate. I suggest to add more substance to each points maybe?

Reply:
I agree about the pointing to the appendix. See the idea for a small table in a later comment. I think that can fit here:)

Reply:
The intention of this section is to connect the relatable concerns of readers to relevant sections of the article, so perhaps I should just provide explicit section names instead of nutshell answers that were derived from the said later sections.